### PR TITLE
Highlight dependencies which were removed in 2.1

### PR DESCRIPTION
**Pull request by ghost**
Based on @janvorli's comment: https://github.com/dotnet/coreclr/issues/18097#issuecomment-391457564 and as stated in https://github.com/dotnet/source-build/blob/0648dba/Documentation/boostrap-new-os.md#prerequisites.

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?branch=pr-en-us-5569&tabs=netcore2x)

Edited by: @jralexander 05/24/18 - added internal review URL.

### CODE REVIEW

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -60,30 +60,36 @@ The following are intended to be examples. The exact versions and names may vary
...
-* libunwind8
 * liblttng-ust0
 * libcurl3
 * libssl1.0.0
-* libuuid1
 * libkrb5-3
 * zlib1g
 * libicu52 (for 14.x)
 * libicu55 (for 16.x)
 * libicu57 (for 17.x)
```
Comment (janvorli) [inline docs/core/linux-prerequisites.md]: Since you are touching this file, could you please also add libicu60 (for 18.x) here?

Comment (kasper3) [reply docs/core/linux-prerequisites.md]: Added below this line.

**Review by janvorli — APPROVED** [review]
LGTM, thank you!

**Review by JRAlexander — APPROVED** [review]
LGTM. Thanks, @kasper3, for your contribution! I'll approve this and it should be live in the next few days.

**Pull request merged**